dm/loader/lightning.go

```diff
@@ -308,7 +309,7 @@ func GetTaskInfoSchemaName(dmMetaSchema, taskName string) string {
 }
 
 // GetLightningConfig returns the lightning task config for the lightning global config and DM subtask config.
-func GetLightningConfig(globalCfg *lcfg.GlobalConfig, subtaskCfg *config.SubTaskConfig) (*lcfg.Config, error) {
+func GetLightningConfig(globalCfg *lcfg.GlobalConfig, subtaskCfg *config.SubTaskConfig, workerName string) (*lcfg.Config, error) {
```

Comment:
task name can be get from subtaskCfg.Name, so we don't need to change the signature

Reply:
I will check if the `subtaskCfg.Name`  can be unchanged during the whole job process.

Reply:
`subtaskCfg.Name` is the `jobID`, I will fix it.